analogwp-templates.php

```diff
@@ -126,19 +126,44 @@ function analog_fail_load() {
 	if ( ! function_exists( 'get_plugins' ) ) {
 		require_once ABSPATH . 'wp-admin/includes/plugin.php';
 	}
-	$file_path         = 'elementor/elementor.php';
-	$installed_plugins = get_plugins();
-	$elementor         = isset( $installed_plugins[ $file_path ] );
+	$file_path = 'elementor/elementor.php';
 
-	if ( $elementor ) {
+	if ( version_compare( get_bloginfo( 'version' ), '5.5a', 'gt' ) ) {
+		$request  = new WP_REST_Request( 'GET', '/wp/v2/plugins/elementor/elementor' );
+		$response = rest_do_request( $request );
+
+		if ( $response->is_error() ) {
+			wp_die( '<p>' . esc_html__( 'An error occurred while checking Elementor is Installed/Activated', 'ang' ) . '</p>' );
+		}
+
+		$server = rest_get_server();
+		$data   = $server->response_to_data( $response, false );
+
+		if ( ! empty( $data['status'] ) && 'inactive' === $data['status'] ) {
+			$is_not_activated = true;
+		} elseif ( ! empty( $data['data']['status'] ) && 404 === $data['data']['status'] ) {
+			$is_not_installed = true;
+		}
+	} else {
+		$installed_plugins = get_plugins();
+		$elementor         = isset( $installed_plugins[ $file_path ] );
+
+		if ( $elementor ) {
+			$is_not_activated = true;
+		} elseif ( ! $elementor ) {
+			$is_not_installed = true;
+		}
+	}
+
+	if ( isset( $is_not_activated ) && $is_not_activated ) {
 		if ( ! current_user_can( 'activate_plugins' ) ) {
 			return;
 		}
 
 		$activation_url = wp_nonce_url( 'plugins.php?action=activate&amp;plugin=' . $file_path . '&amp;plugin_status=all&amp;paged=1&amp;s', 'activate-plugin_' . $file_path );
 		$message        = '<p>' . __( 'Style Kits is not working because you need to activate the Elementor plugin.', 'ang' ) . '</p>';
 		$message       .= '<p>' . sprintf( '<a href="%s" class="button-primary">%s</a>', $activation_url, __( 'Activate Elementor Now', 'ang' ) ) . '</p>';
-	} else {
+	} elseif ( isset( $is_not_installed ) && $is_not_installed ) {
```

Comment:
Assigned `$is_not_installed ` false values on top, then `isset` won't be needed.

Reply:
Assigned default value `false`  for the variable `$is_not_installed`